src/server/debug.rs

```diff
@@ -437,8 +437,8 @@ impl<ER: RaftEngine> Debugger<ER> {
                     v1!(
                         "thread {}: started on range [{}, {})",
                         thread_index,
-                        hex::encode_upper(&start_key),
-                        hex::encode_upper(&end_key)
+                        tikv_util::hex_encode_upper(&start_key),
```

Comment:
Need to be redacted.

Reply:
I thought `debug.rs` would not be run in production... I'll change this soon.

Reply:
Perhaps `diagnostic.rs` is a better name. :(